### PR TITLE
French cheat sheet

### PR DESCRIPTION
**Pull request by bbraithwaite**
https://duck.co/ia/view/french_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@bbraithwaite Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

While the team and the community review your submission, we'd like to make sure you look good! If you'd like a free DuckDuckGo tshirt, please send over the following info to merch@duckduckgo.com, and be sure to put "Dev Merch" in the subject line:

Name:
Street Address:
City:
State / Province:
Country:
Phone (Non-US Orders only): 
T-shirt size:
GitHub username:
Link to this Pull Request:

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by gautamkrishnar** [comment]
:+1:

File: share/goodie/cheat_sheets/json/language/french.json [hunk 1]
```diff
@@ -0,0 +1,178 @@
+{
+    "id": "french_cheat_sheet",
+    "name": "French Cheat Sheet",
+    "description": "Basic French words and phrases",
+    "metadata": {
+        "sourceName": "Linguanaut",
+        "sourceUrl": "http://www.linguanaut.com/english_french.htm"
+    },
+    "aliases": ["french phrases"],
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/french.json]: @bbraithwaite I think we can add more aliases.
you can check other languages' cheat sheets for reference

File: share/goodie/cheat_sheets/json/language/french.json [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+            "key": "Hello"
+        }, {
+            "val": "Bienvenue",
+            "key": "Welcome"
+        }, {
+            "val": "Comment vous appelez-vous?",
+            "key": "What's your name?"
+        }],
+        "Travelling": [{
+            "val": "Tournez à gauche.",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/french.json]: @bbraithwaite lets remove the period at end of val field

File: share/goodie/cheat_sheets/json/language/french.json [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+            "key": "Turn left."
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/french.json]: @bbraithwaite here also

File: share/goodie/cheat_sheets/json/language/french.json (hunk 2 above)
Comment (bbraithwaite) [inline share/goodie/cheat_sheets/json/language/french.json]: I removed the full stops. I've assumed that the comment about correcting following fields was in relation to full stops also?

Comment (jophab) [inline share/goodie/cheat_sheets/json/language/french.json]: @bbraithwaite yes that was :+1:

File: share/goodie/cheat_sheets/json/language/french.json [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+            "key": "Good morning"
+        }, {
+            "val": "Bonsoir",
+            "key": "Good evening"
+        }, {
+            "val": "Bonne Nuit",
+            "key": "Good night"
+        }, {
+            "val": "Je suis désolé",
+            "key": "I'm sorry"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/language/french.json]: @bbraithwaite Maybe "I am sorry" rather than "I'm sorry"

Comment (bbraithwaite) [inline share/goodie/cheat_sheets/json/language/french.json]: @GuiltyDolphin It seems most of the existing language cheat sheets use "I'm sorry". What's the reason for suggesting the change? https://github.com/duckduckgo/zeroclickinfo-goodies/search?utf8=✓&q=I+am+sorry

Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/language/french.json]: @bbraithwaite I thought "I am sorry" is a more direct translation - the "I'm" contraction just felt a bit out of place!

As in "I'm sorry... I didn't catch that", versus "Oh my, I am very sorry!".

It does seem quite a few use it though - it's your call.

Comment (bbraithwaite) [inline share/goodie/cheat_sheets/json/language/french.json]: I see your point. I made the change, anyway.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=french cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/french_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=french cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/french_cheat_sheet_mobile.png)

**Comment by moollaza** [comment]
@bbraithwaite this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit another Instant Answer! :wink:

**Pull request merged**